### DISCUSSION TITLE
Welcome to gripmock Discussions!

### DISCUSSION THREAD

**Discussion by rez1dent3** [bavix/gripmock#111] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gagarina6794** [comment]
Hi gripmock Team,

I stuck with mocking 
// Server streaming: Receive multiple messages from server
How my yml mock should be look like if I expect 5 messages in return?
I have a real service where I call once that external service and I receive a few GetItemsResponse items in one response
rpc GetItems(GetItemsRequest) returns (stream GetItemsResponse);


Thanks in advance,
Iryna

**Reply by rez1dent3** [reply]
Hello. Nothing, but the functionality is in my work. When the opportunity arises, I will write in this thread. I am developing this project in my free time and I am the only one.

**Reply by rez1dent3** [reply]
An anticipatory response. There is also only one client stream. There is no way to send data piecemeal.

**Reply by gagarina6794** [reply]
Thank you for the fast reply, got it, will wait next project improvments :+1:

**Reply by gagarina6794** [reply]
Hi, just wanted to inform that I did my own mock grpc service with Java that covers all my needs.

**Reply by rez1dent3** [reply]
Only now dynamic stabs and support for client streaming, server streaming, bidirectional streaming have appeared. I have long wanted to implement dynamic stabilizers, vacation came and I did it.

GripMock [v3.4.0](https://github.com/bavix/gripmock/releases/tag/v3.4.0)
